CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
 #Copyright (c) 2022 Ultimaker B.V.
 #CuraEngine is released under the terms of the AGPLv3 or higher.
 
+cmake_policy(SET CMP0091 NEW)
```

Comment:
What is a 'CMP0091'? Can I eat it? Google tells me that it has something to do with Windows/MSVC flags; should this be in an if statement for win-builds only?

Reply:
True it is a specific enabling of the new standard CMake policy how MD/MT and MDd/MTd flags are handled on Windows. It is ignored on other operating systems so no `if` needed. 
I personally don't feel a need to comment this. But I'm fine if you want a comment it